tools/webdriver/webdriver/bidi/modules/script.py

```diff
@@ -78,11 +78,11 @@ def __init__(self, context: str, sandbox: Optional[str] = None):
 class SerializationOptions(Dict[str, Any]):
     def __init__(
             self,
-            max_dom_depth: Optional[int] = None,
+            max_dom_depth: Optional[Union[int, None]] = 0,
```

Comment:
Shouldn't this be `Union[int, Undefined]`? That would match with the `browsingContext.setViewport` use case. (And yes we should finalize https://github.com/w3c/webdriver-bidi/issues/642).

Reply:
I didn't know there was an exception to the status quo. Will update it to match that.